### ISSUE TITLE
[Frontend] Create Projects View

### ISSUE THREAD

**Issue by realslimshanky** [UdacityMobileWebScholarship/showcase-app#41]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
None

**Expected behavior:**
 
- A new view for `Projects` is required which will showcase all the projects submitted by users which are available to view and install (or use).
- Project uses [Create React App](https://github.com/facebook/create-react-app) for [React](https://reactjs.org/) and [Material-UI](http://www.material-ui.com/) to produce the frontend.
- The design will be based on the mockup shown below which was chosen from #5 
-  Below is just the mobile view, the actual code should be responsive to support desktop as well.

![image](https://user-images.githubusercontent.com/10004530/39721869-6e441ec6-525e-11e8-983c-e4699f1239d7.png)

**Comment by anurag-majumdar** [comment]
I would like to work on this page.

**Comment by realslimshanky** [comment]
@anurag-majumdar please go ahead and raise a PR. I'm assigning this issue to you. By what time should we expect this feature?

**Comment by realslimshanky** [comment]
**Updates**
@anurag-majumdar according to https://github.com/UdacityMobileWebScholarship/showcase-app/issues/46#issuecomment-387523484, we are currently underway component development for the layout, after this issue is resolved you are free to start working.

**Comment by anurag-majumdar** [comment]
Exactly, I think first I need to work with reusable components required along with blenderskool and divyam and then will finish this page asap. This page can be finished within a few hours after the the initial setup is done.
